packages/gestalt/src/Touchable.css

```diff
@@ -41,3 +44,20 @@
 .grabbing {
   composes: grabbing from "./Cursor.css";
 }
+
+.pressScale {
+  transform: scale(0.99);
+  transform-origin: center;
+}
+
+.pressBackground {
+  composes: lightGrayBg from "./Colors.css";
+}
+
+.pressShadow {
+  box-shadow: 0 0 0 1px #767676;
```

Comment:
@jackhsu978 could you check with AJ / Ashley to see what the defaults should be here?

Reply:
this is what Duplo uses today. will check with them.

Reply:
Here is one of the existing use cases: mweb searchbox shows black outline on focus
<img width="903" alt="Screenshot 2020-06-09 12 43 35" src="https://user-images.githubusercontent.com/5341184/84195295-2a7c0600-aa53-11ea-9d76-6374fa10710a.png">